staging/operator-lifecycle-manager/pkg/controller/operators/olm/downstream_csv_namespace_labeler_plugin.go

```diff
@@ -0,0 +1,212 @@
+package olm
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/operators/internal/pruning"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/kubestate"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/queueinformer"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/informers"
+	v12 "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+)
+
+const labelSyncerLabelKey = "security.openshift.io/scc.podSecurityLabelSync"
+
+// systemNSSyncExemptions is the list of namespaces deployed by an OpenShift install
+// payload, as retrieved by listing the namespaces after a successful installation
+// IMPORTANT: The Namespace openshift-operators must be an exception to this rule
+// since it is used by OCP/OLM users to install their Operator bundle solutions.
+var systemNSSyncExemptions = sets.NewString(
```

Comment:
I take it this is the exact same list as the label syncer's? Any chance we could get them to put that somewhere we could pull in? If not, how do we keep this up-to-date?

Reply:
It is - I just took it from their code for now. But yet, we definitely need to collapse into a single source of truth.